web_graph_improved/__manifest__.py

```diff
@@ -27,7 +27,7 @@
     'author': "Leonardo Donelli,Odoo Community Association (OCA)",
     'license': 'AGPL-3',
     'website': 'http://learts.glaucus.in',
```

Comment:
@legalsylvain URL should be `https://github.com/OCA/web`

Reply:
@nikul-serpentcs, this module is uninstallable. The code will not work, and the module is not available in the app store.
So, this is not relevant to fix the code, and it is not the objective of this PR.

thanks.